python/dgl/graphbolt/internal/sample_utils.py

```diff
@@ -394,4 +463,20 @@ def compact_csc_format(
                     + offset
                 ),
             )
+            if use_timestamp:
+                src_timestamps[src_type] = torch.cat(
```

Comment:
A little confused about this logic, can you add a comment for clafity?

Reply:
Sure. This is to append the timestamps of sampled neighbors to `src_timestamps`. And `_broadcast_timestamps` broadcast destination timestamps to sources.